test/transform/CompatibilityTransform.test.js

```diff
@@ -0,0 +1,13 @@
+import assert from 'assert'
+import domino from 'domino'
+import pagelib from '../../build/wikimedia-page-library-transform'
+
+describe('CompatibilityTransform', () => {
+  it('.enableSupport()', () => {
+    const document = domino.createDocument()
+    pagelib.CompatibilityTransform.enableSupport(document)
+    for (const clazz of Object.values(pagelib.CompatibilityTransform.COMPATIBILITY)) {
```

Comment:
Class is a keyword in ES6 just like C++. When I replace `clazz` with `class`, I get a parsing error in ESLint. Similarly, `thiz` or `that` is also frequently used to refer to a reference of `this`.